builder/bundles.go

```diff
@@ -352,30 +353,32 @@ func resolvePackages(numWorkers int, set bundleSet, packagerCmd []string, emptyD
 			for p := range bundle.AllPackages {
 				queryString = append(queryString, p)
 			}
-			bundle.AllRpmPackages = make(map[string]bool)
+			bundle.AllRpmPackages = make(map[string][]string)
 			// ignore error from the --assumeno install. It is an error every time because
 			// --assumeno forces the install command to "abort" and return a non-zero exit
 			// status. This exit status is 1, which is the same as any other dnf install
 			// error. Fortunately if this is a different error than we expect, it should
 			// fail in the actual install to the full chroot.
 			outBuf, _ := helpers.RunCommandOutputEnv(queryString[0], queryString[1:], []string{"LC_ALL=en_US.UTF-8"})
-
 			rpm, fullRpm := repoPkgFromNoopInstall(outBuf.String())
```

Comment:
Still reviewing. But to get started, can you see if we can use a single dnf query as suggested here https://github.com/clearlinux/mixer-tools/issues/658, instead of having multiple queries (install and repoquery). If yes, we could get rid of the function repoPkgFromNoopInstall().